en/plugins.rst

```diff
@@ -81,8 +86,8 @@ should be like::
 
     "psr-4": {
         (...)
-        "AcmeCorp\\Users\\": "./plugins/AcmeCorp/Users/src",
-        "AcmeCorp\\Users\\Test\\": "./plugins/AcmeCorp/Users/tests"
+        "AcmeCorp\\Users\\": "./src/Plugin/AcmeCorp/Users/src",
```

Comment:
Isn't it a bit odd to have two `src` directories in the path? Using `/Plugin` would only require one.